packages/perseus-core/src/feature-flags.ts

```diff
@@ -7,7 +7,21 @@
  * 2. Do cleanup unused feature flags
  * 3. When no active feature flag is available value will be:
  *    export const PerseusFeatureFlags = [];
+ * 4. Also update the testing/feature-flags-util.ts for testing purpose
  */
-const PerseusFeatureFlags = ["new-radio-widget"] as const;
+const PerseusFeatureFlags = [
+    "new-radio-widget", // TODO(LEMS-2994): cleanup feature flag
+    "image-widget-upgrade", // TODO(LEMS-3521): cleanup feature flag
+] as const;
 
 export default PerseusFeatureFlags;
+
+/**
+ * Checks if a feature flag is enabled in the given props.
+ */
+export function isFeatureOn(
+    props: {apiOptions?: {flags?: Record<string, boolean>}},
+    flag: string,
+): boolean {
+    return props.apiOptions?.flags?.[flag] ?? false;
+}
```

Comment:
What do you think about moving the `getFeatureFlags` function to this file?

And then you can import it from `@khanacademy/perseus-core` rather than doing
```
import {getFeatureFlags} from "../../../../../../testing/feature-flags-util";
```

Reply:
Suggestion: Can you test `isFeatureOn` and `getFeatureFlags`? You could make a `feature-flag.test.ts` file in the same folder as `feature-flag.ts`.